app/styles/modules/_input-row.scss

```diff
@@ -65,9 +65,9 @@
     }
 
     html[dir='rtl'] & {
-      direction: rtl;
+      text-align: right;
+      direction: ltr;
       padding: 0  $input-left-right-padding 0 0;
-      unicode-bidi: bidi-override;
```

Comment:
nit: wont this cause a regression with the special characters? IIRC that was the reason this was even added in the first place @vladikoff